src/components/Menu/Menu.module.css

```diff
@@ -0,0 +1,30 @@
+/*------------------------------------*\
+    # MENU
+\*------------------------------------*/
+
+/**
+ * Menu
+ */
+.menu {
+  position: relative;
+}
+
+.menu__popover {
+  max-width: var(--eds-l-max-width);
+  margin-top: var(--eds-size-1);
+  position: absolute;
+}
+
+.menu__button {
+  background-color: var(--light-color-button-background);
+}
+
+/* // is getting overridden by a style in clickablestyle */
+.menu__button--icon {
```

Comment:
That's annoying. does it only apply to the things which are using clickable style (since it has `>` rules)? anything else forcing this pattern?

With that in mind, and for this case, it actually appears to need ***three*** of these since the descendant selector 
 and `svg` breaks the tie on the two classes being used on `ClickableStyle` :( [0,3,0] vs [0,2,1] I'll do for now, but worth discussing more (separately)

Reply:
>does it only apply to the things which are using clickable style (since it has > rules)?

That's the one I've noticed coming up recently. Maybe we should look into lower the specificity in `ClickableStyle` to make these overrides easier. Though I don't necessarily want it to be easy to override them because I would prefer it if buttons were consistent wherever possible.. 🤔